docs/09.visual-regression-testing.md

```diff
@@ -0,0 +1,67 @@
+## Visual regression testing
+
+Visual regression tests for Storybook's stories are ran in [Sauce Labs](https://saucelabs.com) in order to have consistent display throughout.
+
+### Requirements
+
+In order to run tests locally, you will need:
+
+- [Sauce Labs account](https://app.saucelabs.com/signup)
+- [Sauce Connect](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy)
+
+### Environment variables
+
+Please set the following:
+
+```sh
+export SAUCE_USERNAME=
+export SAUCE_ACCESS_KEY=
+export TEST_BROWSER=
+export ECL_SYSTEM=
+```
+
+### Build storybooks
+
+```sh
+$ yarn dist:storybook
+```
+
+This will build all systems' storybooks and store them in a `dist` folder.
+
+When running tests locally, this `dist` folder will need to be served and exposed to Sauce Labs's cloud in order for its driver to be able to see components.
+
+### Running tests
+
+Use [Sauce Connect runner](https://www.npmjs.com/package/sauceconnect-runner) (or Sauce Connect directly) to open a tunnel from your local environment to Sauce Lab's virtualized browsers in the cloud:
+
+```sh
+$ sc-run yarn test:visual
+```
+
+If you run the test suite several times in a raw and you start getting hard-to-debug differences in snapshots/screenshots, ensure to clear `jest`'s cache:
+
+```sh
+$ jest --clearCache
+```
+
+### Updating references
+
+```sh
+$ yarn test:visual-update
```

Comment:
shouldn't it use `sc-run` too?

Reply:
I've intentionally left it out, as the user can either use `sc` or `sc-run`, I don't want to put an opinion on that.